### PR TITLE
Add English README.

### PR DESCRIPTION
**Pull request by gguillotte**
- Adds a basic English README.
- Adds a link to the English README from the main Japanese README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## これは何？
 
+- [English README](README.en.txt)
```
Comment (hfiguiere) [inline README.md]: Didn't you mean README.en.md ?

**Comment by gguillotte** [comment]
Fixed the link per @hfiguiere and squashed commits.

**Comment by mmuman** [comment]
Hi, regardless the origin of the software, it is traditional to have English first for READMEs (and no, I'm not english native, I could probably write a French one… from the English one :p), why not swap them and have the Japanese one linked from the main one?

**Comment by gguillotte** [comment]
It doesn't matter to me, and I'd rather leave a decision like that to the maintainers. If that's preferred, feel free to close this PR out.

**Pull request merged**